zipkin-lens/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "jest --ci --reporters=jest-standard-reporter",
     "test-dev": "jest",
     "coverage": "jest --coverage",
-    "lint": "eslint src",
+    "lint": "eslint 'src/**/*.{js,jsx}'",
```

Comment:
I'm sorry, I made a mistake setting the lint command... 😢

Reply:
happens to us all!